zavod/zavod/helpers/positions.py

```diff
@@ -24,10 +74,16 @@ def make_position(
     source_url: Optional[str] = None,
     lang: Optional[str] = None,
     id_hash_prefix: Optional[str] = None,
-) -> Entity:
-    """Create consistent position entities. Help make sure the same position
-    from different sources will end up with the same id, while different positions
-    don't end up overriding each other.
+    is_pep: Optional[bool] = True,
+) -> Optional[Entity]:
+    """Creates and returns a Position entity if is_pep is True or if it's
+    confirmed to be a PEP position in the positions database. Otherwise returns
+    None.
+
+    Also ensures the position exists in the positions database for further
+    categorisation.
+
+    Topics in the positions database override topics supplied to this helper.
```

Comment:
actually need to keep the annotation topics separate so that the enricher can add them with the correct provenance